intro-to-tests/intro-to-pytest.md

```diff
@@ -75,15 +75,15 @@ When we use `pytest`, we will be creating more files. In order to anticipate mor
 
 Take care to notice the following:
 
-1. In our project directory, there are 0 files, and 2 folders: `projectname` and `tests`.
+1. In our project directory, there are 0 files, and 3 folders: `projectname`, `tests`, and `venv`.
    - We will replace the folder name `projectname` with our project's name
-1. In our `projectname` folder, there is a file named `somefile.py`
+2. In our `projectname` folder, there is a file named `somefile.py`
    - This represents any Python files that include the functions we're writing
-1. In our `tests` folder, there is a file named `test_somefile.py`
+3. In our `tests` folder, there is a file named `test_somefile.py`
    - This represents a test file that is responsible for testing `projectname/somefile.py`
 1. There are some other files such as `__init__.py`
```

Comment:
Since the others bullets moved to sequential numbers, I think we need to update the number here as well for the order to flow right.
```suggestion
4. There are some other files such as `__init__.py`
```

Reply:
Oops! Let me fix that!